Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,44 @@
+// const process = require('process');
+// console.log(calculate(process.argv[2]));
+
+function calculate(argument,extraOperators) {
+    const invalidArgumentError = new Error('The argument ' + argument + ' is invalid');
```

Comment:
I meant not this but the sign on which error happened. For example:
`"3 2 4 -" - The argument "3" is invalid`